e2e/mock-api/src/app/responses.js

```diff
@@ -63,6 +63,20 @@ export const createStepUpUrl = (url) => {
   return redirectUrl.toString();
 };
 
+export const createStepUpHeader = (url) => {
+  // Grab the client's desired AM URL
+  const referer = new URL(url);
+  const amUrl = referer.searchParams.get('amUrl') || AM_URL;
+
+  // Base 64 of {"TransactionConditionAdvice":["39dfdd15-59a3-473c-a7fc-ecda3bbc3bc8"]}
+  const advices =
+    'eyJUcmFuc2FjdGlvbkNvbmRpdGlvbkFkdmljZSI6WyIzOWRmZGQxNS01OWEzLTQ3M2MtYTdmYy1lY2RhM2JiYzNiYzgiXX0=';
+  const format = '1';
```

Comment:
So far in my live testing I have only seen "format=1" - any idea what this means?